docs/debugging.md

```diff
@@ -0,0 +1,40 @@
+## Debugging Tutorial
+
+### For debugging purposes the following must be installed :
+
+[Visual Studio Code](https://code.visualstudio.com/)
+[Node.js](https://www.npmjs.com/) - to setup node.js you may follow the instructions in [Environment Setup](https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md)
+
+### Setting up VS Code and Launching Telescope :
+
+To begin debugging one must retrieve the launch.json file.
```

Comment:
I'm not sure what this means.  We already have a `launch.json` file in the repo.